cargo-dist/templates/ci/github/release.yml.j2

```diff
@@ -24,12 +24,12 @@
 
 name: Release
 
+{{%- if root_permissions %}}
 permissions:
-  contents: write
-  {{%- if github_attestations %}}
```

Comment:
We still need an `if github_attestations` here, or to add these to the root permissions if attestations are on.